tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1750,13 +1764,24 @@ class ParameterizedOpConverterTestBase
   ParameterizedOpConverterTestBase()
       : trt_mode_(std::get<0>(GetParam())),
         tf_type_(std::get<1>(GetParam())),
-        converter_precision_(std::get<2>(GetParam())) {}
+        converter_precision_(std::get<2>(GetParam())) {
+    LOG(INFO) <<  "%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%";
+    LOG(INFO) <<  "tf_type_: " << DebugString(tf_type_);
+    LOG(INFO) <<  "trt_mode_: " << DebugString(trt_mode_);
+    LOG(INFO) <<  "converter_precision_: " << DebugString(converter_precision_);
+    LOG(INFO) <<  "%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%";
```

Comment:
Can we delete the extra space between "<<" and """? You have two spaces there, and one is enough.

Reply:
Fixed